### DISCUSSION TITLE
Contribution to project

### DISCUSSION THREAD

**Discussion by shreyash-Pandey-Katni** [blue-yonder/tsfresh#1053] [Ideas]
Hello Team, I am interested in contributing to this project. I have gone through the issues but couldn't find any open issue that is recent. Can you please assign me some issues or any enhancements that I can work on? Thank you.

**Comment by nils-braun** [comment]
That is really great @shreyash-Pandey-Katni ! We welcome any contribution to the project - thanks for spending your time helping the community. As you see from the long delay in my answer, I can not spent as much time as I want for tsfresh. Therefore, most of the work is currently answering user support requests.

When it comes to good first issues, how about this one: https://github.com/blue-yonder/tsfresh/issues/992?

**Reply by nils-braun** [reply]
For an issue which will require more code changes, there is https://github.com/blue-yonder/tsfresh/discussions/1037#discussioncomment-6652385

**Comment by shreyash-Pandey-Katni** [comment]
Thank you @nils-braun, I will look into both of them.

**Reply by nils-braun** [reply]
Great! If you need any help, please reach out to me!
